### PR TITLE
Improve gtag script

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments below.

### CODE REVIEW

File: src/js/scripts.js [hunk 1]
```diff
@@ -6,15 +6,19 @@ if (
...
-  const script = document.createElement('script');
-  script.async = true;
-  script.src = `https://www.googletagmanager.com/gtag/js?id=${id}`;
-  document.head.append(script);
+  function gtag() {
+    window.dataLayer.push(arguments);
+  }
+
+  const cone_gtag_id = 'G-VG8YZ0GRD3';
+  const cone_gtag_script = document.createElement('script');
```
Comment (szepeviktor) [inline src/js/scripts.js:14]: These are global variables on the page.
(`coneGtagID`?)

File: src/js/scripts.js [hunk 2]
```diff
@@ -6,15 +6,19 @@ if (
...
+  cone_gtag_script.async = true;
+  document.head.append(cone_gtag_script);
 
   window.dataLayer = window.dataLayer || [];
-  function gtag() { dataLayer.push(arguments); } // eslint-disable-line
-  gtag('js', new Date());
 
-  gtag('config', 'G-VG8YZ0GRD3');
+  gtag('js', new Date());
+  gtag('config', cone_gtag_id);
```
Comment (szepeviktor) [inline src/js/scripts.js:23]: Use the variable!

File: src/js/scripts.js [hunk 3]
```diff
@@ -6,15 +6,19 @@ if (
   issetCookie('cone-cookie-law-analytics')
   && getCookie('cone-cookie-law-analytics') === 'accepted'
```
Comment (szepeviktor) [inline src/js/scripts.js:7]: isset may be unnecessary, get returns null when key is not found
https://github.com/conedevelopment/cone-site/blob/66b616bb81ee51ae72559a6be125b26a13e125e9/src/js/cookie.js#L36-L40

**Comment by adamlaki** [comment]
Thanks Viktor!

File: src/js/scripts.js (hunk 3 above)
Comment (adamlaki) [reply src/js/scripts.js:7]: You are right Viktor! Thanks!

**Pull request merged**